docker-compose.yaml

```diff
@@ -0,0 +1,33 @@
+services:
+  app:
+    build:
+      context: .
+    env_file:
+      - .env
+    ports:
+      - "8000:8000"
+    volumes:
+      - ./:/app
+      - my_media:/vol/web/media
+    command: >
+      sh -c "
+      python manage.py wait_for_db &&
+      python manage.py makemigrations &&
```

Comment:
Running `makemigrations` automatically in the entry command can lead to unexpected migrations being created. It's generally better to handle migrations manually to ensure they are reviewed and tested before applying.